daemon/src/maker_cfd.rs

```diff
@@ -194,7 +194,7 @@ where
         taker_id: Identity,
         ctx: &mut xtra::Context<Self>,
     ) -> Result<()> {
-        tracing::info!(%order_id, "Received proposal from taker {taker_id}");
+        tracing::info!(%order_id, %taker_id,  "Received rollover proposal from taker");
```

Comment:
I think this one is duplicated with the protocol actor.

Reply:
Can't find it.

Reply:
https://github.com/itchysats/itchysats/blob/6e4d2da18c2bd29528fd9379223880d66a153519/daemon/src/rollover_maker.rs#L236-L239